docs/release-notes/known-issues.md

```diff
@@ -30,5 +30,148 @@ As a workaround, and to prevent crashes, Beats will ignore any filters provided
 
 % **Resolved**
 % On [Month/Day/Year], this issue was resolved.
+:::
 
-:::
\ No newline at end of file
+:::{dropdown} Beats panic on restart when "restart_on_cert_change" is enabled on Linux
+
+**Details**
+A known issue was discovered where Beats running on Linux with `restart_on_cert_change` enabled panic during a restart. This occurs because the default seccomp policy does not include the `eventfd2` syscall, which is used by Go runtime versions 1.23.0. While the initial launch is successful, subsequent restarts fail as the seccomp policy is already active, blocking the required syscall.
+
+This affects versions v8.16.6+, v8.17.3+, v8.18.0+, v8.19.0+, v9.0.0+, and v9.1.0+.
```

Comment:
This is also something discussed in Slack earlier. I think it makes sense to make this more prominent (this is what we've done in recent known issues in the Fleet/EA docs).

```suggestion
**Applies to**: v8.16.6+, v8.17.3+, v8.18.0+, v8.19.0+, v9.0.0+, and v9.1.0+

**Details**
A known issue was discovered where Beats running on Linux with `restart_on_cert_change` enabled panic during a restart. This occurs because the default seccomp policy does not include the `eventfd2` syscall, which is used by Go runtime versions 1.23.0. While the initial launch is successful, subsequent restarts fail as the seccomp policy is already active, blocking the required syscall.
```